d20/Manual/Facts/Fields.py

```diff
@@ -1,6 +1,10 @@
 from collections.abc import Iterable
 from inspect import Parameter
 
+from typing import List, Optional, Union, TYPE_CHECKING
+if TYPE_CHECKING:
+    from d20.Manual.Facts import Fact
```

Comment:
adjust the white space and group imports by kind

eg:
```
from inspect import Parameter
from collections.abc import Iterable
from typing import List, Optional, Union, TYPE_CHECKING

if TYPE_CHECKING:
    from d20.Manual.Facts import Fact
```

so builtins are first and there is space before the local imports even as it is wrapped in the conditional from typing